cogs/helpers.py

```
@@ -510,6 +510,31 @@ def __init__(self, bot):
         except ValueError:
             ctx.send("Input must be integers")
 
+    @commands.command()
+    async def food(self, ctx, *args):
+        """Posts a food sale in #foodspotting.
+        Use: `?food Samosas in leacock`
+        You can also attach one picture to your message"""
+        message = " ".join(args)
+        channel = utils.get(self.bot.get_guild(
+            self.bot.config.server_id).text_channels,
+                            name=self.bot.config.food_spotting_channel)
+        username = ctx.message.author
+        pfp = ctx.message.author.avatar_url
+        embed = discord.Embed()
+        try:
+            embed.set_image(url=ctx.message.attachments[0].url)
+        except:
+            pass
```

thank you for not leaving a bare `except`

---

wait jk that *is* a bare except. you can instead put

```python
except Exception:
    pass:
```